tests/conftest.py

```diff
@@ -146,3 +146,37 @@ def scheduler(request):
         yield cluster
     else:
         yield not_cluster
+
+
+@pytest.fixture
+def medium_size_regression():
+    """X, y pair for regression with N >> p.
+
+    There are many more samples in this problem than there are
+    features. Useful for testing stability of solutions.
+    """
+    X, y = make_regression(
+        chunks=100, n_samples=500, n_features=100, n_informative=10, random_state=0
```

Comment:
```suggestion
        chunks=100, n_samples=500, n_features=100, n_informative=100, random_state=0
```

This seems like a better idea without regularization. It also wouldn't be a bad idea to reduce the number of features.